obspy/__init__.py

```diff
@@ -27,10 +27,7 @@
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-from future.builtins import *  # NOQA
-from future.utils import PY2, native_str
+from __future__ import absolute_import, division, print_function
```

Comment:
Do we need any `__future__` imports at all? Seems obsolete?

https://docs.python.org/3.5/reference/simple_stmts.html#future
```
The features recognized by Python 3.0 are absolute_import, division, generators,
unicode_literals, print_function, nested_scopes and with_statement. They are all
redundant because they are always enabled, and only kept for backwards compatibility.
```